pkg/proc/proc_test.go

```diff
@@ -4103,7 +4049,7 @@ func TestNextUnknownInstr(t *testing.T) {
 
 func TestReadDeferArgs(t *testing.T) {
 	if runtime.GOARCH == "arm64" {
-		t.Skip("arm64 does not support Stacktrace for now")
+		t.Skip("arm64 does not support ReadDeferArgs for now")
```

Comment:
Can you do an experiment for me?
In stack.go there's a function called EvalScope with receiver *Defer, inside this function there's a line that says:

```
scope.Regs.Regs[scope.Regs.SPRegNum].Uint64Val = uint64(scope.Regs.CFA - int64(bi.Arch.PtrSize()))
```

can you change it to:

```
switch bi.Arch.(type) {
case *AMD64:
     scope.Regs.Regs[scope.Regs.SPRegNum].Uint64Val = uint64(scope.Regs.CFA - int64(bi.Arch.PtrSize()))
case *ARM64:
     scope.Regs.Regs[scope.Regs.SPRegNum].Uint64Val = uint64(scope.Regs.CFA)
}
```

And then see if the test passes.
I'm not sure that's the problem but it seems like it would be the most likely cause.

Reply:
emm, no any change(even the err value):
```sh
=== RUN   TestReadDeferArgs
--- FAIL: TestReadDeferArgs (9.68s)
    proc_test.go:4095: value of argument 'a' at frame 1, deferred call 1: 0 (expected 42)
    proc_test.go:4099: value of argument 'b' at frame 1, deferred call 1: 0 (expected 61)
    proc_test.go:4095: value of argument 'a' at frame 2, deferred call 2: 72 (expected 1)
    proc_test.go:4099: value of argument 'b' at frame 2, deferred call 2: 64 (expected -1)
FAIL
FAIL    github.com/go-delve/delve/pkg/proc      10.516s
FAIL
```

Reply:
Ok, then I don't know, but the problem should be somewhere in that function.